packages/core/upload/admin/src/components/Breadcrumbs/Breadcrumbs.js

```diff
@@ -19,13 +20,21 @@ export const Breadcrumbs = ({ breadcrumbs, ...props }) => (
               .splice(index + 1, breadcrumbs.length - 1)
               .map(parent => parent.id)}
             key={`breadcrumb-${crumb?.id ?? 'menu'}`}
+            currentFolderId={currentFolderId}
+            onChangeFolder={onChangeFolder}
           />
         );
       }
 
-      if (crumb.href) {
+      if (crumb.href !== undefined) {
         return (
-          <CrumbLink key={`breadcrumb-${crumb?.id ?? 'root'}`} as={NavLink} to={crumb.href}>
+          <CrumbLink
+            key={`breadcrumb-${crumb?.id ?? 'root'}`}
+            as={onChangeFolder ? 'button' : NavLink}
```

Comment:
If the type is `button` we'd also need to add `type=button`, otherwise it would be `type=submit` which can have side effects.